Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Use the official ROS 2 humble base image
+FROM ros:humble
+
+# Set the working directory
+WORKDIR /ros2_workspace/src
+
+COPY . .
+# Install additional dependencies
+RUN apt-get update \
+    && apt-get -y --quiet --no-install-recommends install \
+    gcc \
+    git \
+    libxml2-dev \
+    libxslt-dev \
+    python3 \
+    ros-humble-ament-cmake-clang-format\
+    ros-humble-rviz2
+
+# Set environment variables
+ENV ROS_DISTRO humble
+ENV ROS_VERSION 2
+WORKDIR /ros2_workspace
+RUN . /opt/ros/humble/setup.sh
+RUN rosdep update
+RUN rosdep install --from-paths src --ignore-src -y
+
+RUN . /opt/ros/humble/setup.sh && colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-up-to grid_map_geo
+
+# Source the ROS2 setup file
+RUN echo "source /ros2_workspace/install/setup.bash" >> ~/.bashrc
+# Expose the default ROS 2 communication ports
+EXPOSE 11311
+
+# Run a default command, e.g., starting a bash shell
+CMD ["bash"]
```

Comment:
Add missing newline please so you can `cat` the file in terminal without issues.

Reply:
I dont understand what you mean.

Reply:
See how when you type `cat Dockerfile`, the last line overlaps with my terminal?

That's because it's missing a trailing newline. Github warns about it. All you need to do is open the file at the end, and hit enter (newline) and save.


![image](https://github.com/ethz-asl/grid_map_geo/assets/25047695/b019ad76-7cb0-4a3d-b032-79ddecd40e1f)

Reply:
I got it . Thanks